src/gui/navigationpanehelper.cpp

```diff
@@ -86,9 +86,6 @@ void NavigationPaneHelper::updateCloudStorageRegistry()
         // We currently don't distinguish between new and existing CLSIDs, if it's there we just
         // save over it. We at least need to update the tile in case we are suddently using multiple accounts.
         foreach (Folder *folder, _folderMan->map()) {
-            if (folder->vfs().mode() == Vfs::WindowsCfApi) {
```

Comment:
Not sure about this one.
In the upstream commit, there is the following comment: `// we might need to add or remove the panel entry as cfapi brings this feature out of the box`.
I think that we should investigate.

Reply:
In fact that's a very good call. The "cfapi brings this feature out of the box" is kind of... well... depends where you consider cfapi stops. Things is it got published with other APIs which are related to file emblems and how the registration was done in the navigation pane and so the cfapi examples conflate everything together.

I think it's because on the owncloud side they went full steam using all of those extra APIs. And I'd argue it's not the best place to do it as that hole being plugged with a special case shows. Personally I'd go with status quo for now (using the older APIs for navigation pane and emblems) and then if there's really a need switch the navigation pane and emblems integration to the new APIs for all sync folders (I suspect this is doable but I'm not sure).

I'd avoid at all cost to carry the maintenance burden of two different way to do those integrations on Windows depending on the type of sync folder you got...

Reply:
Yes, I agree about the maintenance problem of having two different ways (VFS and classical). That was the point I wanted to check before deciding.
Thanks for the info